student.rb

```diff
@@ -0,0 +1,25 @@
+require './person'
+
+# Create Student Class
+class Student < Person
+  attr_reader :classroom, :parent_permission
+
+  # Create constructor
+  def initialize(_age, _name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    super(@age, @name, @parent_permission)
+    @classroom = classroom
+  end
+
+  # Create classroom
+  def classrooms(classroom)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Great job working on the student belongs to a classroom association. However, the method seems to have been "half worked on". For this method to be complete, it should look like something like this:

```
def classroom=(classroom)
      @classroom = classroom
          classroom.student.push(self) unless classroom.student.includes?(self)
  end
```

![image](https://user-images.githubusercontent.com/86318284/199956618-f36e0ad7-f623-4f2d-9aab-12a550d141e6.png)